portfolio.js

```diff
@@ -55,8 +55,238 @@ links.forEach((link) => {
+    image: 'images/component_1.png',
+    imageDesktop: 'images/component_1_.png',
+    alt: 'Project number one',
+    keywords: [
+      'HTML/CSS',
+      'Ruby on Rails',
+      'Javascript'],
+    liveVersion: '#',
+    source: '#',
+
+  },
+  {
+    id: 2,
+    class: 'two',
+    classDesktop: 'pc-left',
+    classImgDesktop: 'pc-left',
+    boxClass: 'boxDsk2',
+    name: 'Project name goes here',
+    description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nis. Ut aliquip ex ea commodo consequat. </br>  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi.',
+    image: 'images/component_2.png',
+    imageDesktop: 'images/pc_img_left.png',
+    alt: 'Project number two',
+    keywords: [
+      'HTML/CSS',
+      'Ruby on Rails',
+      'Javascript'],
+    liveVersion: '#',
+    source: '#',
+
+  },
+  {
+    id: 3,
+    class: 'three',
+    classDesktop: 'pc-left-bigger',
+    classImgDesktop: 'pc-left-bigger',
+    boxClass: 'boxDsk3',
+    name: 'Project name goes here',
+    description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nis. Ut aliquip ex ea commodo consequat. </br>  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi.',
+    image: 'images/component_3.png',
+    imageDesktop: 'images/pc_img_left_2.png',
+    alt: 'Project number three',
+    keywords: [
+      'HTML/CSS',
+      'Ruby on Rails',
+      'Javascript'],
+    liveVersion: '#',
+    source: '#',
+
+  },
+  {
+    id: 4,
+    class: 'four',
+    classDesktop: 'pc-right-yellow',
+    classImgDesktop: 'pc-left',
+    boxClass: 'boxDsk4',
+    name: 'Project name goes here',
+    description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nis. Ut aliquip ex ea commodo consequat. </br>  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi.',
+    image: 'images/component_4.png',
+    imageDesktop: 'images/pc_right.png',
+    alt: 'Project number four',
+    keywords: [
+      'HTML/CSS',
+      'Ruby on Rails',
+      'Javascript'],
+    liveVersion: '#',
+    source: '#',
+
+  },
```

Comment:
- [ ] As requested by the first reviewer you should work on responsiveness as there is horizontal scrolling on the project so when clicking the buttons then it has same issue on pop-up window as well.
### Responsiveness 
![his](https://user-images.githubusercontent.com/81727988/211178042-1c50f6d3-5723-481f-9c1f-f83d0905c0ba.png)